packages/preview-middleware/src/base/flp.ts

```diff
@@ -21,6 +21,34 @@ const DEVELOPER_MODE_CONFIG = new Map([
     ['xx-viewCache', 'false']
 ]);
 
+/**
+ * SAPUI5 delivered namespaces from https://ui5.sap.com/#/api/sap
+ */
+const UI5_LIBS = [
```

Comment:
I think it is fine to keep it here for now, but if required by any other tool we might consider moving this information to `@sap-ux/ui5-info`

Reply:
Thanks Klaus, I will keep this in mind.